functional/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jmailen.kotlinter'
+    id 'java-test-fixtures'
```

Comment:
I decided to give a try to the test fixtures plugin to emphasize that all the sources here are meant to be used in tests sources only.